lib/Runtime/Library/IntlEngineInterfaceExtensionObject.cpp

```diff
@@ -298,6 +304,53 @@ namespace Js
             return instance;
         }
     };
+
+    template<typename TResource>
+    class FinalizableICUObject : public FinalizableObject
+    {
+    public:
+        typedef void (__cdecl *CleanupFunc)(TResource *);
+
+    private:
+        FieldNoBarrier(TResource) *resource;
+        FieldNoBarrier(CleanupFunc) cleanupFunc;
+
+    public:
+        FinalizableICUObject(TResource *resource, CleanupFunc cleanupFunc) :
+            resource(resource),
+            cleanupFunc(cleanupFunc)
+        {
+
+        }
```

Comment:
redundant blank lines throughout this class

Reply:
That was ~intentional, I think it looks nicer than having the braces on separate lines without a gap or the same line?